src/main/kotlin/frc/robot/lib/MarkerCommands.kt

```diff
@@ -0,0 +1,22 @@
+package frc.robot.lib
+
+import edu.wpi.first.wpilibj.Alert
+import edu.wpi.first.wpilibj.DataLogManager
+import edu.wpi.first.wpilibj2.command.Command
+import edu.wpi.first.wpilibj2.command.Commands
+import org.littletonrobotics.junction.AutoLogOutput
+import org.littletonrobotics.junction.Logger
+
+const val ABNORMAL_EVENT_NAME = "PROBLEM!"
```

Comment:
```suggestion
const val ABNORMAL_EVENT_NAME = "### [Error] DRIVER PROBLEM REPORTED ###"
```
Reasons for the suggestions:
1. It's important to include "error"/"warning" in the message so AdvantageScope will syntax-highlight it:
   ![console-1-6f9327aee7a933a520ad94b347941150.png](https://github.com/user-attachments/assets/aaa01654-7b38-4ac9-8ac6-a443920ee293)
2. The message should be a bit more descriptive and visible.